src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/requests/makerecalldecisions/UpdateRecommendationForNoRecallRequest.kt

```diff
@@ -44,7 +26,6 @@ fun updateRecommendationForNoRecallRequest(status: Status = Status.DRAFT) = """
   },
   "responseToProbation": "They have not responded well",
   "whatLedToRecall": "Increasingly violent behaviour",
```

Comment:
whatLedToRecall doesn't apply to 'no recall'

Reply:
done